app/views/turns/form.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
You can use a default layout for common views in order to avoid repeating the HTML structure

Reply:
Ok, I'm going to apply the HTML structure